chain-tx-enclave/tx-validation/make.sh

```diff
@@ -2,5 +2,5 @@
 set -e
 
 source /root/.docker_bashrc
-cargo build -p tx-validation-app
+cargo build -p chain-abci
```

Comment:
I think this file and `tx-query/make.sh` are not used anywhere.

Reply:
I haven't verified -- I think originally they were used in those docker-compose-based integration tests... not sure if "getting started" Makefile uses them